stable/mattermost-team-edition/values.yaml

```diff
@@ -0,0 +1,113 @@
+# Default values for mattermost-team-edition.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+image:
+  repository: mattermost/mattermost-team-edition
+  tag: 5.2.1
+  imagePullPolicy: Always
+
+## Enable persistence using Persistent Volume Claims
+## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
+## ref: https://docs.gitlab.com/ee/install/requirements.html#storage
+##
+persistence:
+  ## This volume persists generated data from users, like images, attachments...
+  ##
+  data:
+    enabled: true
+    size: 10Gi
+    ## If defined, volume.beta.kubernetes.io/storage-class: <storageClass>
+    ## Default: volume.alpha.kubernetes.io/storage-class: default
+    ##
+    # storageClass:
+    accessMode: ReadWriteOnce
```

Comment:
What if I run multiple replicas? How would Mattermost behave?

Reply:
For team edition, you can run just one instance.

Reply:
made note in the Readme.md as limitation.